astroquery/ned/tests/data/query_references.xml

```diff
@@ -14,7 +14,7 @@ under cooperative agreement AST0122449 with The Johns Hopkins University.
 </DESCRIPTION>
```

Comment:
OK, great, thanks.  Maybe amend the commit message to include "NED now returns valid XML" or something along those lines?  Astroquery has actually lead to a few upstream fixes of this sort in the past, in Vizier at least, not sure about NED...